docs/guides/operations-guide/index.md

```diff
@@ -5,6 +5,47 @@ sidebar_position: 30
 
 # Operations Guide
 
+## General usage of osism commands
```

Comment:
Move this into the manager/index.md file.

Reply:
Done, also beautified the image of on that page a bit.